documentation/book/con-what-the-cluster-operator-does.adoc

```diff
@@ -1,34 +1,27 @@
 
 [id='con-what-the-cluster-operator-does-{context}']
 = Overview of the Cluster Operator component
 
-The Cluster Operator is in charge of deploying a Kafka cluster alongside a Zookeeper ensemble.
-As part of the Kafka cluster, it can also deploy the topic operator which provides operator-style topic management via `KafkaTopic` custom resources.
-The Cluster Operator is also able to deploy a Kafka Connect cluster which connects to an existing Kafka cluster.
-On OpenShift such a cluster can be deployed using the Source2Image feature, providing an easy way of including more connectors.
+The Cluster Operator is responsible for deploying a Kafka cluster alongside a Zookeeper ensemble.
 
-.Example architecture for the Cluster Operator
+To deploy a Kafka cluster:
 
-image:cluster-operator.png[Cluster Operator]
+* A `Kafka` resource with the cluster configuration is created within the Kubernetes cluster.
+* The Cluster Operator deploys a corresponding Kafka cluster, based on what is declared in the `Kafka` resource.
 
-When the Cluster Operator is up, it starts to _watch_ for certain Kubernetes resources containing the desired Kafka, Kafka Connect, or Kafka Mirror Maker cluster configuration.
-By default, it watches only in the same namespace where it is installed.
-The Cluster Operator can be configured to watch for more namespaces.
-Cluster Operator watches the following resources:
+For more information on the configuration options supported by the `Kafka` resource, see xref:assembly-deployment-configuration-kafka-str[].
 
-* A `Kafka` resource for the Kafka cluster.
-* A `KafkaConnect` resource for the Kafka Connect cluster.
-* A `KafkaConnectS2I` resource for the Kafka Connect cluster with Source2Image support.
-* A `KafkaMirrorMaker` resource for the Kafka Mirror Maker instance.
+The Cluster Operator can also deploy:
 
-When a new `Kafka`, `KafkaConnect`, `KafkaConnectS2I`, or `Kafka Mirror Maker` resource is created in the Kubernetes cluster, the operator gets the cluster description from the desired resource and starts creating a new Kafka, Kafka Connect, or Kafka Mirror Maker cluster by creating the necessary other Kubernetes resources, such as StatefulSets, Services, ConfigMaps, and so on.
+* A Topic Operator to provide operator-style topic management via `KafkaTopic` custom resources.
+* A Kafka Connect cluster to connect to an existing Kafka cluster.
 
-Every time the desired resource is updated by the user, the operator performs corresponding updates on the Kubernetes resources which make up the Kafka, Kafka Connect, or Kafka Mirror Maker cluster.
-Resources are either patched or deleted and then re-created in order to make the Kafka, Kafka Connect, or Kafka Mirror Maker cluster reflect the state of the desired cluster resource.
-This might cause a rolling update which might lead to service disruption.
+NOTE: On OpenShift, a Kafka Connect deployment can incorporate a Source2Image feature to provides a convenient way to include connectors.
```

Comment:
Here are missing both Kafka Mirror Maker and Kafka Bridge.

Reply:
Updated, as with the other intro